Jasmine/Common/ViewModels/LevelSelectViewModelProtocol.swift

```diff
@@ -0,0 +1,27 @@
+/// The ViewModel for the level selector part.
+protocol LevelSelectorViewModelProtocol {
+    /// The defaults levels in the game.
+    var defaultLevels: [GameInfo] { get }
+    /// The custom levels in the game.
+    var customLevels: [GameInfo] { get }
+
+    /// Get all the phrases from the given level ID.
+    ///
+    /// - Parameter levelID: the level ID
+    /// - Returns: all phrases in the level ID
+    func getLevelPhrases(from levelID: Int) -> [String]
```

Comment:
hmm if that works. Should still return `Phrases` because the `PhrasesExplorerViewModel` requires that if I want to segue to `PhrasesExplorerVC`.